src/applications/my-education-benefits/containers/ConfirmationPage.jsx

```diff
@@ -259,78 +260,92 @@ const loadingPage = (
+  // });
 
-    const confirmationResult = 'pending';
+  const { submission, data } = form;
+  const { response } = submission;
+  const name = data.veteranFullName;
 
-    switch (confirmationResult) {
-      case 'approved': {
-        return approvedPage;
-      }
-      case 'denied': {
-        return deniedPage;
-      }
-      case 'pending': {
-        return pendingPage;
-      }
-      case 'loading': {
-        return loadingPage;
-      }
-      default: {
-        return (
-          <div>
-            <h3 className="confirmation-page-title">Claim received</h3>
-            <p>
-              We usually process claims within <strong>a week</strong>.
-            </p>
-            <p>
-              We may contact you for more information or documents.
-              <br />
-              <i>Please print this page for your records.</i>
-            </p>
-            <div className="inset">
-              <h4>
-                My Education Benefits Claim{' '}
-                <span className="additional">(Form 22-1990)</span>
-              </h4>
-              {name ? (
-                <span>
-                  for {name.first} {name.middle} {name.last} {name.suffix}
-                </span>
-              ) : null}
+  const confirmationResult = claimStatus || 'loading';
 
-              {response ? (
-                <ul className="claim-list">
-                  <li>
-                    <strong>Date received</strong>
-                    <br />
-                    <span>
-                      {moment(response.timestamp).format('MMM D, YYYY')}
-                    </span>
-                  </li>
-                </ul>
-              ) : null}
-            </div>
+  switch (confirmationResult) {
+    case 'approved': {
+      return approvedPage;
+    }
+    case 'denied': {
+      return deniedPage;
+    }
+    case 'pending': {
+      return pendingPage;
+    }
+    case 'loading': {
+      return loadingPage;
+    }
+    default: {
+      return (
+        <div>
+          <h3 className="confirmation-page-title">Claim received</h3>
+          <p>
+            We usually process claims within <strong>a week</strong>.
+          </p>
+          <p>
+            We may contact you for more information or documents.
+            <br />
+            <i>Please print this page for your records.</i>
+          </p>
+          <div className="inset">
+            <h4>
+              My Education Benefits Claim{' '}
+              <span className="additional">(Form 22-1990)</span>
+            </h4>
+            {name ? (
+              <span>
+                for {name.first} {name.middle} {name.last} {name.suffix}
+              </span>
+            ) : null}
+
+            {response ? (
+              <ul className="claim-list">
+                <li>
+                  <strong>Date received</strong>
+                  <br />
+                  <span>
+                    {moment(response.timestamp).format('MMM D, YYYY')}
```

Comment:
Is it possible to use https://date-fns.org/ for this? We're trying to get moment out of the system because it's a deprecated library